packages/block-library/src/pullquote/edit.js

```diff
@@ -77,21 +69,39 @@ class PullQuoteEdit extends Component {
 		}
 	}
 
-	pullQuoteTextColorSetter( colorValue ) {
-		const { setTextColor } = this.props;
+	function pullQuoteTextColorSetter( colorValue ) {
 		setTextColor( colorValue );
 		this.wasTextColorAutomaticallyComputed = false;
```

Comment:
Shouuld we use a ref for this?

Reply:
Yes, looks like I forgot some of `this.*` :)